Backend_Application/src/main/java/dsd/cohort/application/ingredient/IngredientController.java

```diff
@@ -0,0 +1,24 @@
+package dsd.cohort.application.ingredient;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/ingredients")
+public class IngredientController {
+
+    private final IngredientService ingredientService;
+
+    public IngredientController(IngredientService ingredientService) {
+        this.ingredientService = ingredientService;
+    }
+
+    @GetMapping("/{id}")
```

Comment:
Are you wanting to stick to our generated id value for brevity? I know we talked about keeping the foodId from the API as well.

Reply:
I did add the food id from the API, do we want to use that as your unique id? I'm ok either way.

Reply:
IMO using the foodId for the DTO(data transfer object) when communicating data to the FE should be the way to go.